MainQuickView.py

```diff
@@ -0,0 +1,28 @@
+from workspacemanager.WorkspaceManagerQuickView import WorkspaceManagerQuickView
+from powderprojection.PowderProjectionQuickView import PowderProjectionQuickView
+from MainPresenter import MainPresenter
+import workspacemanager,powderprojection
+
+#TODO Error in logging framework, FIX/ASK !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+class MainQuickView():
```

Comment:
What is MainQuickView?. I know, but someone else won't. Docstring needed at the very least.